apps/app/src/views/components/TablePage.tsx

```diff
@@ -104,20 +106,23 @@ export const TablePage = (props: TablePageProps) => {
 
   return (
     <Box
+      w="full"
       bg="bg-surface"
       boxShadow={{ base: 'none', md: useColorModeValue('sm', 'sm-dark') }}
       borderRadius={useBreakpointValue({ base: 'lg', md: 'lg' })}
       ref={scrollableContainerRef}
     >
       <Stack spacing="5">
         <Box px={{ base: '4', md: '6' }} pt="5">
-          <Stack direction={{ base: 'column', md: 'row' }} justify="space-between">
+          <Stack
+            direction={{ base: 'column', md: `row${ctaRight ? '-reverse' : ''}` }}
+            justify="space-between"
+          >
             {ctaRouteColorTextUndefined ? null : (
               <Button
                 as={ctaOnClick ? undefined : RouterLink}
                 to={ctaRoute || ''}
                 onClick={ctaOnClick}
-                leftIcon={<FiPlus fontSize="1.25rem" />}
```

Comment:
double-checking that this is meant to affect prescription/order tables too

Reply:
🤷 I kinda hate the `+` tbh

Reply:
maybe double-check w/ jomi?